common-content/en/module/logic/search/index.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "1000 Bottles of Wine"
+time = 60
+emoji= "🐀"
+[build]
+render = 'never'
+list = 'local'
+publishResources = false
++++
+
+Now we're going to solve a puzzle together. We're going to think through a few different ways to approach it. This is a well-known puzzle and you may certainly google the answer. However, you will steal from yourself the opportunity of learning this important idea, so maybe _don't_ do that.
```

Comment:
```suggestion
Now we're going to solve a puzzle together. We're going to think through a few different ways to approach it. This is a well-known puzzle and it's certainly possible to google the answer. However, we'd suggest: don't do that - it would steal from yourself the opportunity of learning this important idea for yourself.
```